doc/developer-guide.rst

```diff
@@ -0,0 +1,46 @@
+.. py:currentmodule:: lsst.ts.ATDome
+
+.. _lsst.ts.ATDome.developer_guide:
+
+###############
+Developer Guide
+###############
+
+The ATDome CSC is implemented using `ts_salobj <https://github.com/lsst-ts/ts_salobj>`_.
+
+The CSC controls the dome using a TCP/IP connection to a low-level controller provided by Astronomical Consulting Equipment.
```

Comment:
Unfortunately I do not know where manual from the vendor lives. Do you? (note that the manual is riddled with errors, but it is still better than nothing).

Reply:
in this collection are the electrical and smartDome manuals. Probably both should be linked since when someone wants to modify the labVIEW code they'll want the electrical diagram
https://docushare.lsst.org/docushare/dsweb/View/Collection-6331

Reply:
Great! Thank you very much!